.github/workflows/build_wheels.yml

```diff
@@ -21,7 +21,7 @@ jobs:
       fail-fast: false
       matrix:
         python-version: ["3.9"]
-        os: [ubuntu-22.04, macos-13, windows-2019]
+        os: [ubuntu-22.04]  # , macos-13, windows-2019]
```

Comment:
For now wheels are only going to work on Linux, and they aren't proper `manylinux` wheels yet anyway.  But it's close.  This will be my next major effort.